### PR TITLE
Added html for section 2

### LINKED ISSUE #3: html section 2
We need to create the layout of the section 2

### PR DESCRIPTION
**Pull request by Tim007007**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
 </head>
```
Comment (Evbgsl) [inline index.html]: Delete empty lines

File: index.html [hunk 2]
```diff
@@ -22,19 +23,25 @@ <h1 class="download__title">Peaceful shopping mindful money</h1>
...
+        <h2 class="warranty__title">1-year warranty boost</h2>
+        <p class="warranty__copy">ZEN cards have a very neat feature that will help you whenever your electronic devicesstart failing after the standard warranty ends, which is pretty common.</p>
+        <p class="warranty__copy">When you buy any electronics with your ZEN shopping Mastercard, their warranty getsextended by one year. How does it work? It simply does. Automatically.</p>
+      </div>
+      <div class="warranty__image-block">
+        <img src="./assets/images/2.jpg" alt="Basket">
+      </div>
+    </div>
+  </section>
+  <!-- секция 3тест  -->
```
Comment (Evbgsl) [inline index.html]: Delete test

File: index.html [hunk 3]
```diff
@@ -22,19 +23,25 @@ <h1 class="download__title">Peaceful shopping mindful money</h1>
     </div>
     <div class="download__image-block"> <img src="./assets/images/1.jpg" alt="Bag and device"></div>
```
Comment (Evbgsl) [inline index.html]: use new line after open tag
it s recommended to close a single tag Bag and device

File: index.html [hunk 4]
```diff
@@ -22,19 +23,25 @@ <h1 class="download__title">Peaceful shopping mindful money</h1>
...
   </section>
-  <!-- секция 2  -->
-  <section>Секция 2</section>
-  <!-- секция 3  -->
+  <!-- секция 2 -->
+  <section class="warranty" id="warranty">
+    <div class="warranty__wrapper">
+      <div class="warranty__text-block">
+        <h2 class="warranty__title">1-year warranty boost</h2>
+        <p class="warranty__copy">ZEN cards have a very neat feature that will help you whenever your electronic devicesstart failing after the standard warranty ends, which is pretty common.</p>
```
Comment (Evbgsl) [inline index.html]: use new line for tag

File: index.html [hunk 5]
```diff
@@ -22,19 +23,25 @@ <h1 class="download__title">Peaceful shopping mindful money</h1>
...
+        <p class="warranty__copy">When you buy any electronics with your ZEN shopping Mastercard, their warranty getsextended by one year. How does it work? It simply does. Automatically.</p>
+      </div>
+      <div class="warranty__image-block">
+        <img src="./assets/images/2.jpg" alt="Basket">
```
Comment (Evbgsl) [inline index.html]: use / for single tag

File: index.html (hunk 4 above)
Comment (Evbgsl) [inline index.html]: split a long line into paragraphs

**Review by Evbgsl — APPROVED** [review]
Need to resolve conflict?

**Review by KoloskovaAlla — APPROVED** [review]

**Pull request merged**